source/Module/Rule.ProcessMitigation/Convert/Methods.ps1

```diff
@@ -127,6 +127,47 @@ function Get-MitigationPolicyToEnable
     }
 }
 
+function Get-MitigationPolicyToDisable
```

Comment:
lets put a online help comment block with what this function is for: (Synopsis) at least.